packages/showcases/core/csf/components/structure/accordion.csf.js

```diff
@@ -41,4 +41,14 @@ export const argTypes = {
       type: 'boolean',
     },
   },
+  onOpen: {
+    type: { name: 'function', required: false },
+    description: 'Callback fired when the accordion is opening',
+    action: 'onOpen',
+  },
+  onClose: {
+    type: { name: 'function', required: false },
+    description: 'Callback fired when the accordion is closing',
+    action: 'onClose',
+  },
```

Comment:
Should not be set here but only for react. Svelte don't need to have onOpen input etc ... 

Please check `VtmnQuantity` for example.

And add `open` and `close` into handles array on the csf file.